src/Tizen.NUI/src/internal/BackKeyManager.cs

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright(c) 2019 Samsung Electronics Co., Ltd.
```

Comment:
2019 to 2020.
and could you add some simple sample code or sample app to figure out how this can be used? ^^;

Reply:
DONE